.eslintrc.js

```diff
@@ -19,12 +19,16 @@ module.exports = {
         remarkable: false,
         sinon: false,
     },
+    parser: '@babel/eslint-parser',
     rules: {
         'class-methods-use-this': 0, // fixme
+        'flowtype/no-types-missing-file-annotation': 0,
```

Comment:
We should try to avoid disabling any linting rules. Let's chat about the need for these in the team channel.

Reply:
Sure, here's some more context for this error: https://github.com/gajus/eslint-plugin-flowtype/issues/225

Reply:
We don't use flow in this repo, though. Should we remove the flow linting entirely? The original comment applies to the other overrides, as well.

Reply:
As discussed on Slack, I've kept the flowtype rule disabling where it currently is and moved the rest of the rule disabling into the overrides (targeted specifically to TS files)!